src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -701,6 +853,15 @@ private static unsafe void ZeroMatrixItemsCore(AlignedArray destination, int c,
             }
         }
 
+        /// <summary>
+        /// Updates span items with threshold.
+        /// </summary>
+        /// <param name="primalUpdate">The primal update value.</param>
+        /// <param name="count">The count of items.</param>
+        /// <param name="source">The source values.</param>
+        /// <param name="threshold">The threshold value.</param>
+        /// <param name="v">The v span.</param>
```

Comment:
Wasn't sure what this was exactly so the param documentation is very simple.

Also, should this be renamed since this file is being updated?